temporian/io/test/polars_test.py

```diff
@@ -0,0 +1,496 @@
+        )
+
+        expected_df = pl.DataFrame(
+            {
+                "product_id": [666964, 666964, 574016],
+                "costs": [740.0, 508.0, 573.0],
+                "timestamp": [1.0, 2.0, 3.0],
+            }
+        )
+
+        df = to_polars(evset)
+        assertEqualDFRandomRowOrderPolars(self, df, expected_df)
+
+    def test_evset_to_df_multiple_index(self):
+        evset = event_set(
+            timestamps=[1.0, 2.0, 3.0, 1.1, 2.1, 3.1, 1.2, 2.2, 3.2],
+            features={
+                "sma_a": [10.0, 10.5, 11.0, 13.0, 13.5, 14.0, 16.0, 16.5, 17.0],
+                "x": ["X1", "X1", "X1", "X2", "X2", "X2", "X2", "X2", "X2"],
+                "y": ["Y1", "Y1", "Y1", "Y1", "Y1", "Y1", "Y2", "Y2", "Y2"],
+            },
+            indexes=["x", "y"],
+        )
+
+        expected_df = pl.DataFrame(
+            {
+                "x": ["X1", "X1", "X1", "X2", "X2", "X2", "X2", "X2", "X2"],
+                "y": ["Y1", "Y1", "Y1", "Y1", "Y1", "Y1", "Y2", "Y2", "Y2"],
+                "sma_a": [10.0, 10.5, 11.0, 13.0, 13.5, 14.0, 16.0, 16.5, 17.0],
+                "timestamp": [1.0, 2.0, 3.0, 1.1, 2.1, 3.1, 1.2, 2.2, 3.2],
+            }
+        )
+
+        df = to_polars(evset)
+        assertEqualDFRandomRowOrderPolars(self, df, expected_df)
+
+    def test_evset_to_df_string_feature(self):
+        evset = event_set(
+            timestamps=[1.0, 2.0, 3.0],
+            features={
+                "product_id": [666964, 666964, 574016],
+                "costs": ["740.0", "508.0", "573.0"],
+            },
+            indexes=["product_id"],
+        )
+
+        expected_df = pl.DataFrame(
+            {
+                "product_id": [666964, 666964, 574016],
+                "costs": ["740.0", "508.0", "573.0"],
+                "timestamp": [1.0, 2.0, 3.0],
+            }
+        )
+
+        df = to_polars(evset)
+        assertEqualDFRandomRowOrderPolars(self, df, expected_df)
+
+    def test_evset_to_df_unix_timestamp(self):
+        evset = event_set(
+            timestamps=[
+                datetime.datetime(2023, 11, 1),
+                datetime.datetime(2023, 11, 2),
+                datetime.datetime(2023, 11, 3),
+            ],
+            features={"f": [1, 2, 3]},
+        )
+
+        # Assuming to_polars correctly handles datetime conversion
+        df = to_polars(evset)
+        self.assertTrue(df["timestamp"].dtype == pl.Datetime)
+        evset2 = from_polars(df)
+        assert evset2.schema.is_unix_timestamp
+
+    def test_timestamps_params(self):
+        evset = event_set(
+            timestamps=[
+                datetime.datetime(2023, 11, 1),
+                datetime.datetime(2023, 11, 2),
+                datetime.datetime(2023, 11, 3),
+            ],
+            features={"f": [1, 2, 3]},
+        )
+
+        df = to_polars(evset)
+        self.assertTrue("timestamp" in df.columns)
+        self.assertTrue(df["timestamp"].dtype == pl.Datetime)
+
```

Comment:
missing testing other values for `timestamps` and `timestamp_to_datetime` params, see pandas_test